sdk/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -258,94 +242,6 @@ public void update(Observable observable, Object arg) {
         }
     }
 
-    /**
-     * Creates a new configuration object based on the provided parameters
-     * will read the API key and other configuration values from the manifest if it is not provided
-     *
-     * @param androidContext         The context of the application
-     * @param apiKey                 The API key to use
-     * @param enableExceptionHandler should we automatically handle uncaught exceptions?
-     * @return The created config
-     */
-    @NonNull
-    private static Configuration createNewConfiguration(@NonNull Context androidContext,
-                                                        String apiKey,
-                                                        boolean enableExceptionHandler) {
-        Context appContext = androidContext.getApplicationContext();
-
-        // Attempt to load API key and other config from AndroidManifest.xml, if not passed in
-        boolean loadFromManifest = TextUtils.isEmpty(apiKey);
-
-        if (loadFromManifest) {
-            try {
-                PackageManager packageManager = appContext.getPackageManager();
-                String packageName = appContext.getPackageName();
-                ApplicationInfo ai =
-                    packageManager.getApplicationInfo(packageName, PackageManager.GET_META_DATA);
-                Bundle data = ai.metaData;
-                apiKey = data.getString(MF_API_KEY);
-            } catch (Exception ignore) {
-                Logger.warn("Bugsnag is unable to read api key from manifest.");
-            }
-        }
-
-        if (apiKey == null) {
-            throw new NullPointerException("You must provide a Bugsnag API key");
-        }
-
-        // Build a configuration object
-        Configuration newConfig = new Configuration(apiKey);
-        newConfig.setEnableExceptionHandler(enableExceptionHandler);
-
-        if (loadFromManifest) {
-            try {
-                PackageManager packageManager = appContext.getPackageManager();
-                String packageName = appContext.getPackageName();
-                ApplicationInfo ai =
-                    packageManager.getApplicationInfo(packageName, PackageManager.GET_META_DATA);
-                Bundle data = ai.metaData;
-                populateConfigFromManifest(newConfig, data);
-            } catch (Exception ignore) {
-                Logger.warn("Bugsnag is unable to read config from manifest.");
-            }
-        }
-        return newConfig;
-    }
-
-    /**
-     * Populates the config with meta-data values supplied from the manifest as a Bundle.
-     *
-     * @param config the config to mutate
-     * @param data   the manifest bundle
-     * @return the updated config
-     */
-    @NonNull
-    static Configuration populateConfigFromManifest(@NonNull Configuration config,
```

Comment:
what's the default access modifier that's applied here? Does moving this have any external implications?

Reply:
The package-level modifier will be applied, so anything with the `com.bugsnag.android` namespace can access this. I don't believe this has any external implications as I'm not aware of any other notifiers which call this method.